docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md

```diff
@@ -88,95 +96,78 @@ terraform plan
 - How to generate a SSH Key.
     - On most systems you can generate a SSH Key with: `ssh-keygen -t rsa -b 4096`
     - This will output where you public key is stored, for example: `~/.ssh/id_rsa.pub`
     - You can get the value with `cat ~/.ssh/id_rsa.pub`
 
-As mentioned above, there are two options for pipeline's destination database. These are Postgres and Snowflake. Your chosen database needs to be specified with `pipeline_db` variable. Allowed values for that variable are `postgres` and `snowflake`. Respective `<destination>.terraform.tfvars` file should be filled in according to the chosen database. Only database specific variables are different in those two tfvars files.
+As mentioned above, there are several options for the pipeline's destination database. These are Postgres, Snowflake, Redshift and Databricks. For each destination there is a dedicated block of variables that will need to be filled and the `<destination>_enabled` variable set to `true`.
 
 ##### Postgres
 
-If you choose Postgres as destination, there is no additional step. Respective variables need to be filled according to the desired setup. Necessary resources like Postgres instance, database, table, user will be created by Pipeline Terraform module.
-
-##### Snowflake
-
-If you choose Snowflake as destination, there is one more additional step. Pipeline Terraform module doesn't create necessary Snowflake resources, unlike Postgres resources. It only deploys the Snowflake Loader. Therefore Snowflake resources need to be created before deploying the pipeline. We've created [another Terraform module](https://github.com/snowplow/quickstart-examples/tree/snowflake-loader/terraform/aws/snowflake) for this purpose. You can follow the Optional Step 2 below to learn how to use Snowflake Terraform module. Snowflake Terraform module will give outputs after it is applied. These outputs need to be passed to `snowflake.terraform.tfvars`.
+If you choose Postgres as destination, there are no additional steps. Respective variables need to be filled according to the desired setup. Necessary resources like a PostgreSQL RDS instance, database, table, user will be created by the Terraform modules.
 
-**Step 2 (Snowflake only): Run Snowflake terraform module**
+It is strongly recommended to:
 
-It is possible to use Snowflake as the destination in AWS pipelines. However necessary resources need to be created in Snowflake before starting the pipeline.
+1. Update the `postgres_db_password` to a new value that only **you** know
+2. Set the `postgres_db_ip_allowlist` to a list of CIDR addresses that will need to access the database - this can be systems like BI Tools, or your home address so you can query the database from your laptop
```

Comment:
```suggestion
1. Update the `postgres_db_password` to a new value that only **you** know
2. Set the `postgres_db_ip_allowlist` to a list of CIDR addresses that will need to access the database - this can be systems like BI Tools, or your local IP address so that you can query the database from your laptop

:::
```

Do we have an example format for the allowlist somewhere?